core/src/fullnode.rs

```diff
@@ -307,6 +316,7 @@ pub fn new_banks_from_blocktree(
         bank_forks_info,
         blocktree,
         ledger_signal_receiver,
+        completed_slots_receiver,
```

Comment:
How big are you going to make this tuple before changing it to a struct with named fields? For me, it's usually the transition from 2 items to 3.

Reply:
We could use a struct, but the return values here will need to be immediately destructured as they are passed to different threads (WindowService + ReplayStage), so I left them as is.